src/calculateRentalCost.js

```diff
@@ -7,7 +7,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
```

Comment:
don't use magic numbers, use constants instead

Reply:
I cant use const because sum might change, depends on if if statement is true. (or maybe i didnt get what you wanted me to do..)

Reply:
The idea is to move number 40 to constants - for example const pricePerDay = 40